_posts/2016-03-14-twis-55.md

```diff
@@ -0,0 +1,38 @@
+---
+layout:     post
+title:      This Week In Servo 55
+date:       2016-03-14 00:30:00
+summary:    What's up with Servo for the week of 07 Mar 2016 - 14 Mar 2016
+categories:
+---
+
+In the [last week](https://github.com/pulls?page=1&q=is%3Apr+is%3Amerged+closed%3A2016-03-07..2016-03-14+user%3Aservo), we landed 78 PRs in the Servo organization's repositories.
+
+I'm pleased to announced that [Aneesh Agrawal](https://github.com/aneeshusa) has become a reviewer! He is a major contributor to the [Saltstack](https://github.com/saltstack/salt/) project, and has been providing contributions and feedback on our CI infrastructure for a long time now. Thanks for all your help, Aneesh!
+
+### Notable Additions
+
+ - mbrubeck [fixed](https://github.com/servo/servo/pull/9969) a case where layout would re-resolve generated content
+ - mrobinson [simplifed](https://github.com/servo/servo/pull/9962) the display list representation
+ - antrik continued to [fix](https://github.com/servo/servo/pull/9948) tricky ipc-channel edge cases
+ - larsberg [added](https://github.com/servo/saltfs/pull/239) support for ARM 32-bit and 64-bit cross-compiles to our CI
+ - kichjang [redesigned](https://github.com/servo/servo/pull/9217) `ScriptMsg`
```

Comment:
I used to consistently use real names in the past, except when one wasn't available. However, this is harder to automate since real names may be on github or as part of their email.